include/rdma/fi_eq.h

```diff
@@ -76,8 +78,13 @@ struct fid_wait {
 };
 
 struct fi_mutex_cond {
+#ifndef _WIN32
 	pthread_mutex_t		*mutex;
 	pthread_cond_t		*cond;
+#else /* _WIN32 */
+	CRITICAL_SECTION	*mutex;
+	CONDITION_VARIABLE	*cond;
+#endif /* _WIN32 */
```

Comment:
the is one major difference between pthread mutexes & windows critical section: pthread mutex may have cross-process attributes, critical section could not.
there is API in windows to implement pthread mutexes/conditional variables: https://msdn.microsoft.com/en-us/library/windows/desktop/ms682052(v=vs.85).aspx